.gitlab-ci.yml

```diff
@@ -0,0 +1,89 @@
+image: hugomrdias/node-alpine:test
```

Comment:
Can we publish this under "ipfs" or "protocol labs" or something? What's the difference between this image and regular `node-alpine`?